src/lib/layouts/DocsTutorial.svelte

```diff
@@ -53,6 +54,20 @@
                     {/if}
                 </ul>
                 <div class="u-cross-center relative flex">
+                    {#if back}
+                        <a
+                            href={back}
+                            class="
+						web-button is-text is-only-icon web-u-cross-center web-u-size-40
+						u-position-absolute u-inset-inline-start-0 -translate-x-1/2"
```

Comment:
Is all this extra spacing intended?

Reply:
I've reverted it from how it was before. I only switch the 100% negative translateX (from pink v1 class) to the Tailwind -translate-x-1/2 so it visually looks as in the design.

Reply:
I meant the indentation in the code itself:

```
                            class="
						web-button is-text is-only-icon web-u-cross-center
						web-is-not-mobile -translate-x-1/2"
```

instead of something like:

```
                            class="web-button is-text is-only-icon web-u-cross-center
					web-is-not-mobile -translate-x-1/2"
```